silabs_flasher/rootfs/etc/s6-overlay/scripts/universal-silabs-flasher-up

```diff
@@ -109,12 +111,18 @@ if bashio::config.true 'verbose'; then
     verbose="-v"
 fi
 
+ezsp_baudrate_arg=""
+if bashio::config.has_value 'ezsp_baudrate'; then
+    ezsp_baudrate_arg="--ezsp-baudrate \"${ezsp_baudrate}\""
```

Comment:
I'd recommend using arrays here, like we have it in other places (e.g. the Matter Server add-on):

```sh
ezsp_baudrate_arg=()
if bashio::config.has_value 'ezsp_baudrate'; then
    ezsp_baudrate_arg("--ezsp-baudrate", ${ezsp_baudrate})
```


```sh
universal-silabs-flasher \
...
    "${ezsp_baudrate_arg[@]}"
...
```